src/BotPlutusInterface/Files.hs

```diff
@@ -257,6 +309,10 @@ unDummyPrivateKey :: DummyPrivKey -> Crypto.XPrv
 unDummyPrivateKey (FromSKey key) = key
 unDummyPrivateKey (FromVKey key) = key
 
+-- | Used as a signing keys for foreign signers BPI has no access to (ie. multisig schemes)
+dummyXPrv :: Crypto.XPrv
+dummyXPrv = generate ("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" :: Base.String) ("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" :: Base.String)
```

Comment:
```suggestion
dummyXPrv = join generate("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" :: Base.String)
```

Reply:
🎖️

Reply:
I wasn't aware join had this behaviour on functions, and im unsure how I feel about it
It feels misleading

Reply:
Yeah, I actually got `join`  from pointfree tool output myself =) I was just concerned a bit about typing exact amount of letters two times in arow - stopped to check if they are of the same length XD

Reply:
I think i'd rather leave as is, I might check if the passphrase input has to be the same length, as if we can change it to just `"pass phrase"`, then this call makes more sense